andela/wg-aces
# #159250142 Add Relation between WorkoutLog and WorkoutSession

#### What does this PR do?
Add a relation between WorkoutLog and WorkoutSession models. 

#### Description of Task to be completed?
This task involves adding a foreign key to link Workout Logs and Workout Sessions

#### How should this be manually tested?
After cloning this repository;
-  Run the following commands
    - `python manage.py migrate`
    - `python manage.py runserver`
    -   Add a new workout with exercises 
    -   Add a new log to that newly created workout
    -   Use a database admin tool like postico or pgAdmin to check the `manager_workoutlog` table rows
    - Newly created logs should have a `session_id` column to relate directly to the `manager_workoutsession` table

#### What are the relevant pivotal tracker stories?
[#159250142](https://www.pivotaltracker.com/story/show/159250142)

#### Screenshots
N/A
